samples/drivers/espi/src/main.c

```diff
@@ -47,10 +47,35 @@ struct oob_response {
 	u8_t buf[MAX_RESP_SIZE];
 };
 
-#ifdef CONFIG_ESPI_GPIO_DEV_NEEDED
-static struct device *gpio_dev0;
-static struct device *gpio_dev1;
 #define PWR_SEQ_TIMEOUT    3000u
+
+/* The devicetree node identifier for the board power rails pins. */
+#define BRD_PWR_NODE DT_INST(0, microchip_mec15xx_board_power)
+
+#ifdef USING_DT_MACROS
```

Comment:
When is this not true?

Reply:
This should always be true and nothing should be done in the else.
However when I use the new DT macros, the sample doesn't compile.
Could you take a look what could be wrong in the node access?